README.md

```diff
@@ -3,6 +3,24 @@
 [Hyperledger Besu](https://github.com/hyperledger/besu/) is a Hyperledger enterprise-grade Java-based,
 Apache 2.0 licensed Ethereum client.
 
+Welcome to the **Besu Documentation** repository.
 The software sources are hosted in https://github.com/hyperledger/besu
 
 This repository only contains the sources for [Besu documentation website hosted by ReadTheDocs](https://besu.hyperledger.org/).
+
+
+## Useful Links
+
+* [Besu Docs](http://besu.hyperledger.org/)
```

Comment:
Use a shared reference-style link here (see for instance https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links or https://stackoverflow.com/questions/45265914/markdown-what-are-inline-and-reference-style-links)